### ISSUE TITLE
Issue sharing link with custom geojson layer

### ISSUE THREAD

**Issue by enriquetuya** [GlobalFishingWatch/map-client#982]
Affects version: `3.0.1`
When saving a workspace a custom geojson layer is not being persisted correctly. 

Steps to reproduce: 
1st open the map `globalfishingwatch.org`
2nd add a custom layer. I've tried this one: https://gist.githubusercontent.com/enriquetuya/4415d69e3abd926f1a5a822a3380093b/raw/981ce222fab9dcb7cde93069d83e94e56b4c8186/map.geojson
3rd check the saved workspace, you will see that the workspace has the following information instead of the correct url: 

```
            {  
               "id":"333741e1-751c-4cbb-8bf1-a71b51c3b4c6",
               "url":"https://api-dot-skytruth-pelagos-production.appspot.com/v2/directory/333741e1-751c-4cbb-8bf1-a71b51c3b4c6/source",
               "title":"TEST",
               "label":"TEST",
               "description":"",
               "type":"Custom",
               "visible":true,
               "opacity":1,
               "added":true,
               "color":"#C0CA33"
            },
```

**Comment by nerik** [comment]
@enriquetuya The workspace URL was fine, but there was a race condition when loading custom layers from the workspace. That's fixed on develop.

**Comment by enriquetuya** [comment]
Fixed and Deployed to production in version `3.0.2`
